tests/unit/store-test.js

```diff
@@ -1157,3 +1157,57 @@ test("find with filter function returns array proxy that has a remove function t
   assert.equal(store._findById("person", 8).get("firstName"), "");
   assert.equal(store._findById("person", 8).get("lastName"), "new");
 });
+
+var Listing;
+
+moduleFor('service:simple-store', "store unit tests -- custom primary key", {
```

Comment:
I wasn't sure about where the best place for these tests was -- glad to reorganize if you'd like.

Also, I can add more tests if you'd like -- just let me know what you think deserves focus.